src/Validators/ApplicantValidator.php

```diff
@@ -44,7 +44,7 @@ public static function validateAdditionalFields(array $applicant): bool
         if (
             (
                 (int)$applicant['upfront'] + (int)$applicant['edaid'] + (int)$applicant['diversitech']
-            ) > Globals::ACADEMYPRICE
+            ) > (int)$applicant['fee']
```

Comment:
what if there is no applicant fee? its an optional field so if its blank, this validation should use the academyprice global instead